mteb/tasks/Classification/est/estonian_valence.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class EstonianValenceClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="EstonianValenceClassification",
+        dataset={
+            "path": "kardosdrur/estonian-valence",
+            "revision": "9157397f05a127b3ac93b93dd88abf1bdf710c22",
+        },
+        description="Dataset containing annotated Estonian news data from the Postimees and Õhtuleht newspapers.",
+        reference="https://figshare.com/articles/dataset/Estonian_Valence_Corpus_Eesti_valentsikorpus/24517054",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["est-Latn"],
+        main_score="accuracy",
+        date=(
+            "1857-01-01",  # Inception of Postimees
+            "2023-11-08",  # dataset publication
+        ),
+        form=["written"],
+        domains=["News"],
+        task_subtypes=["Sentiment/Hate speech"],
+        dialect=None,
```

Comment:
```suggestion
        dialect=[],
```

Reply:
How is this better or different?

Reply:
None == Not annotated, [] means that is was considered and we don't believe there is any dialects.